vassal-app/src/main/java/VASSAL/build/module/RandomTextButton.java

```diff
@@ -43,7 +43,7 @@
  * @author Michael Blumoehr
  * This component places a button into the controls window toolbar.
  * Pressing the button generates random numbers or strings and displays the
- * result in the Chatter
+ * result in the Chatter.REbiu
```

Comment:
Sorry, what is this?

Reply:
Well, the other change is just a newline change. Is this just a placeholder PR or something?